### PR TITLE
Missione patch 1

### PR DESCRIPTION
**Pull request by naruto-4**
Update README

### CODE REVIEW

**Comment by naruto-4** [comment]
what is the issue

File: README.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
 [![Open Source Helpers](https://www.codetriage.com/magento/magento2/badges/users.svg)](https://www.codetriage.com/magento/magento2)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/magento/magento2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/magento-2/localized.svg)](https://crowdin.com/project/magento-2)
-<h2>Welcome</h2>
-Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich eCommerce solution that gets results.
+<h2>What is Magento 2 ?</h2>
```
Comment (jeff-matthews) [inline README.md:4]:
```suggestion
## What is Magento 2?
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+It's an e-commerce platform build in PHP, to provide online merchants with a flexible shopping cart system, as well as control over the look, functionality and content of the online store .It also offers powerful search engines for marketing and catalog management tools.
```
Comment (jeff-matthews) [inline README.md:5]:
```suggestion
It's an e-commerce platform written in PHP that provides online merchants with a flexible shopping cart system as well as control over the look, functionality, and content of the online store. It also offers powerful search engines for marketing and catalog management tools.
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
+<br>
```
Comment (jeff-matthews) [inline README.md:6]: There's no need for these two HMTL line breaks (`<br>`). A single new line is sufficient to separate paragraphs in markdown.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,22 @@
...
+<br>
+<strong>Welcome</strong> to Magento 2 installation! We're glad you chose to install Magento 2
```
Comment (jeff-matthews) [inline README.md:8]:
```suggestion
**Welcome** to your Magento 2 installation! We're glad you chose to install Magento 2.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (naruto-4) [reply README.md:8]: Thank you sir .i made the given corrections.

**Pull request closed without merging**